src/components/draggable/draggable.test.js

```diff
@@ -0,0 +1,135 @@
+import * as React from "react";
+import DraggableContainer from "./draggable-container.component";
+import DraggableItem from "./draggable-item.component";
+import { Checkbox } from "../checkbox";
+import Box from "../box";
+
+import {
+  draggableItem,
+  draggableItemByPosition,
+} from "../../../cypress/locators/draggable";
+
+import CypressMountWithProviders from "../../../cypress/support/component-helper/cypress-mount";
+
+const DraggableCustom = ({ getOrder, ...props }) => {
+  const handleUpdate = (items) => {
+    console.log(items);
+    if (getOrder) {
+      getOrder();
+    }
+  };
+
+  return (
+    <DraggableContainer {...props} getOrder={handleUpdate}>
+      <DraggableItem key="1" id={1}>
+        <Checkbox label="Draggable Label One" />
+      </DraggableItem>
+      <DraggableItem key="2" id={2}>
+        <Checkbox label="Draggable Label Two" />
+      </DraggableItem>
+      <DraggableItem key="3" id={3}>
+        <Checkbox label="Draggable Label Three" />
+      </DraggableItem>
+      <DraggableItem key="4" id={4}>
+        <Checkbox label="Draggable Label Four" />
+      </DraggableItem>
+    </DraggableContainer>
+  );
+};
+
+const DraggableDifferentContainers = ({ getOrder, ...props }) => {
+  const handleUpdate = (items) => {
+    console.log(items);
+    if (getOrder) {
+      getOrder();
+    }
+  };
+
+  return (
+    <Box>
+      <DraggableContainer mb={9} getOrder={handleUpdate} {...props}>
+        <DraggableItem key="4" id={4}>
```

Comment:
Should this first DraggableItem be `key="5"` `id={5}` and the others increment from that if it's the start of a block of different containers?

Reply:
it doesn't matter :) it is just an example for tests :)